content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -516,26 +517,34 @@ work between Windows and Linux:
   node. They should be applied to all containers as a best practice if the operator
   wants to avoid overprovisioning entirely.
 * `securityContext.allowPrivilegeEscalation` -
-   not possible on Windows; none of the capabilities are hooked up
+   not possible on Windows; none of the capabilities are hooked up. Pod API
```

Comment:
There are other places in the doc that talks about these fields under `securityContext`. Shouldn't we also mention it there? Or is that not done for features behind a feature gate?